.github/workflows/ci.yml

```diff
@@ -1,16 +1,15 @@
 name: CI
-
 on:
+  workflow_dispatch:
   pull_request:
+  # Triggering CI on default branch improves caching
+  # see https://docs.github.com/en/free-pro-team@latest/actions/guides/caching-dependencies-to-speed-up-workflows#restrictions-for-accessing-a-cache
   push:
     branches:
       - main
 
-env:
-  isPrFromForkedRepo: ${{ github.event_name == 'pull_request' && github.event.pull_request.head.repo.owner.login != 'guardian' }}
```

Comment:
It's safe to get rid of the `isPrFromForkedRepo` condition - as you can see [in this test PR](https://github.com/guardian/google-search-indexing-observatory/pull/4) by an external account, it says "1 workflow awaiting approval" & "First-time contributors need a maintainer to approve running workflows." - GitHub has had this safeguard in place since [April 2021](https://github.blog/2021-04-22-github-actions-update-helping-maintainers-combat-bad-actors/).

![image](https://user-images.githubusercontent.com/8761239/209845323-15068ce3-e7dd-45f2-aa53-9e72a3021a19.png)